templates/jenkins/files/README.md

```diff
@@ -1,19 +1,24 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
```

Comment:
Jenkins needs a Post-install section (see Drupal 8 as an example), to discuss the initial password business.

Reply:
WIP

Reply:
Done

Reply:
Please, check the post-install